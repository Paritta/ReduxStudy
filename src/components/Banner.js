import React from "react";
import styled from "styled-components";
import background from "../asset/background.jpg"

const BannerBackground = styled.div`
    background: url(${background}) left bottom;
    background-size: cover;
    height: 40vh;
`;

const Banner = () => {
    return (
        <div>
            <BannerBackground />

        </div>
    );
};

export default Banner;
