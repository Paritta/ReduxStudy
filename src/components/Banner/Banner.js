import React from "react";
import styled from "styled-components";
import background from "../../asset/background.jpg"

const BannerBackground = styled.div`
    background: url(${background}) left bottom;
    background-size: cover;
    height: 45vh;
`;

export const Banner = () => {
    return (
        <div>
            <BannerBackground />
        </div>
    );
};

export default Banner;
