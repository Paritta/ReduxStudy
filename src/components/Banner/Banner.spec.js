import React from "react";
import { Banner } from "./Banner";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";

it("Card render correctly", () => {
    const tree = ReactTestRenderer.create(
        <Banner />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
