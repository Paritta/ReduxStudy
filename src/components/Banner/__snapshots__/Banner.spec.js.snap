// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
.c0 {
  background: url(background.jpg) left bottom;
  background-size: cover;
  height: 40vh;
}

<div>
  <div
    className="c0"
  />
</div>
`;
